locale/en/download/package-manager.md

```diff
@@ -164,6 +165,19 @@ Or build manually from pkgsrc:
 cd pkgsrc/lang/nodejs && bmake install
 ```
 
+## n
+
+`n` is a simple to use node version manager for Mac and Linux. Specify the target version to install using a rich syntax, or select from a menu of previously downloaded versions. 
```

Comment:
```suggestion
`n` is a simple to use Node.js version manager for Mac and Linux. Specify the target version to install using a rich syntax, or select from a menu of previously downloaded versions. 
```

We try to use _Node.js_ (and joke that "the js is silent") in text and reserve `node` for the executable.

Reply:
hopefully the dot is silent too

Reply:
Hmm, I will be reviewing my READMEs. I had missed that pattern. 😄

Reply:
(Thanks @Trott for suggestion.)